CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/hierarchical/HierarchicalControllerTopComponent.java

```diff
@@ -717,6 +694,7 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
         }
     }
 
+    @PluginInfo(pluginType = PluginType.UPDATE, tags = {"LOW LEVEL", "MODIFY"})
```

Comment:
@aldebaran30701 I think we can omit the "MODIFY" here. "LOW LEVEL" should be enough for all the state updaters. This is because if I untick "LOW LEVEL" but have "MODIFY" then the state updaters will still show up and that is the main problem.

Reply:
I'm changing this but to note is that it doesn't actually save the state to the graph. It happens as an update and that was my reasoning for the low level, but also does modify the visibility and dimmed attributes of every vertex and transaction.

Reply:
Yeah doing a quick look over the plugin, I think MODIFY rather than LOW LEVEL is the one to keep here

Reply:
Switched to MODIFY